CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Unreleased
 
+## 1.18.0 / 2023-12-22
+
+* [FEATURE] promlint: Allow creation of custom metric validations. #1311
+* [FEATURE] api: Change HTTP Client into interface, so users can use customized clients, e.g. with retries on 429. #1387
+* [BUGFIX] histograms: Add timer to reset ASAP after bucket limiting has happened. #1367
+* [BUGFIX] testutil: Fix comparison of metrics with empty Help strings. #1378
+* [BUGFIX] Fix building client_golang in wasip1 OS. #1350
```

Comment:
I'm not sure if that's considered a bugfix or enhancement 🤔

Reply:
Indeed. We have to ask ourselves: Did we support that before? I think the answer is "no", so it might a feature (:

Reply:
My line of thought for adding it as bugfix was "People already expected support for this before, but only now it is working"

But after searching I can't find many people complaining about this lack of support, so I guess we could add it as feature :)